man/man8/zpoolconcepts.8

```diff
@@ -93,6 +93,64 @@ compromised.
 The minimum number of devices in a raidz group is one more than the number of
 parity disks.
 The recommended number is between 3 and 9 to help increase performance.
+.It Sy draid , draid1 , draid2 , draid3
+A variant of raidz which provides integrated distributed hot spares that allow
+faster resilvering while retaining the benefits of raidz.  A dRAID vdev is
+constructed from multiple internal raidz groups, each with D data devices
+and P parity devices which are distributed over the children.  This allows
+all of the children to participate and for fully sequential resilvering.
+.Pp
+This fixed stripe width significantly effects both usable capacity and IOPs.
+For example, with the default D=8 and 4k disk sectors the minimum allocation
+size is 32k.  If using compression this relatively large allocation size can
+reduce the effective compression ratio.  When using ZFS volumes and dRAID the
+default volblocksize property is increased to account for the allocation
+size. Use of a mirrored
+.Sy special
+vdev type to store small blocks is recommended when using dRAID.
+.Pp
+In regards to IO/s, performance is similar to raidz since for any read all
+D disks must be accessed.  Delivered random IOPs can be reasonably
+approximated as floor(D/<data disks>)*<single-drive-IOPs>.
+.Pp
+Like raidz a draid can have single-, double-, or triple-parity.  The
+.Sy draid1 ,
+.Sy draid2 ,
+and
+.Sy draid3
+types can be used to specify the parity level.
+The
+.Sy draid
+vdev type is an alias for
+.Sy draid1 .
+.Pp
+A draid with N disks of size X, D data disks, P parity level, and S distributed
+hot spares can hold approximately (N-S)*(D/(D+P))*X bytes and can withstand P
+device(s) failing before data integrity is compromised.
```

Comment:
Is this `D` is referring to "The number of data devices per redundancy group", specified by e.g. `draid:8D`?  If so, can you make that more clear (`D data disks per redundancy group`); and if not can you use a different letter or refactor this in terms of `D data disks per redundancy group`?

Reply:
Done, I added "D data disks per redundancy group".